takwimu/takwimu_ui/src/components/CountryContent/index.js

```diff
@@ -144,7 +150,7 @@ class ViewAnalysis extends Component {
               label: classes.selectButtonLabel
             }}
           >
-            Go
+            {countryContent.view_content_action_label}
```

Comment:
Lets have this documented as how we want to localize

Reply:
If we ever get to `localization` then yes, right now, to UI, everything is just content from API.